### DISCUSSION TITLE
Show and tell

### DISCUSSION THREAD

**Discussion by sh1zicus** [sh1zicus/dots-hyprland-old#38] [Show and tell]
Show off something you've made

**Comment by PharmaRacist** [comment]
https://github.com/user-attachments/assets/bc2be619-bd96-4c2f-8d68-cc0f7e75d24f

look at this bar  one of the best thing is in separating every module ❤️

**Reply by nichroot** [reply]
broo that looks so cool!! would be very cool if you are able to modify the install script to install this ❣️

**Reply by sh1zicus** [reply]
> broo that looks so cool!! would be very cool if you are able to modify the install script to install this ❣️

I think there you just need to replace all `sh1zicus` -> `Pharmaracist` in the links to the repository

**Reply by nichroot** [reply]
> > broo that looks so cool!! would be very cool if you are able to modify the install script to install this ❣️
> 
> I think there you just need to replace all `sh1zicus` -> `Pharmaracist` in the links to the repository

oh, which script should i replace the lines in? the install.sh script inside the main repo or the setup.sh script inside the wiki repo?

**Comment by PharmaRacist** [comment]
.
├── bar
│   ├── focus
│   │   ├── workspaces_hyprland.js
│   │   └── workspaces_sway.js
│   ├── main.js
│   └── normal
│       ├── battery.js
│       ├── clock.js
│       ├── MResources.js
│       ├── music.js
│       ├── netspeed.js
│       ├── resources.js
│       ├── spaceleft.js
│       ├── spaceright.js
│       ├── system.js
│       ├── tray.js
│       ├── utils.js
│       ├── wintitle.js
│       ├── workspaces_hyprland.js
│       └── workspaces_sway.js

**Comment by sh1zicus** [comment]
I'm thinking about removing focus mode from the bar

**Reply by PharmaRacist** [reply]
leave it and add more mods for the customizability

**Comment by PharmaRacist** [comment]
look at this .
├── main.js
└── modules
    ├── bar_toggles.js
    ├── battery.js
    ├── cava.js
    ├── clock.js
    ├── kb_layout.js
    ├── mixed.js
    ├── music.js
    ├── powerdraw.js
    ├── resources.js
    ├── simple_button.js
    ├── simple_clock.js
    ├── spaceleft.js
    ├── spaceright.js
    ├── system.js
    ├── tray.js
    ├── utils.js
    ├── window_title.js
    ├── workspaces_hyprland_focus.js
    ├── workspaces_hyprland.js
    ├── workspaces_sway_focus.js
    └── workspaces_sway.js

**Comment by PharmaRacist** [comment]
https://github.com/user-attachments/assets/14e397ba-3df7-475e-93b8-2aef3ed98f86

added more bar modes and spliting more and changing bar fs
- floating
- normal
- minimal
- notched
- hanged
.
├── main.js
└── modules
    ├── bar_toggles.js
    ├── battery.js
    ├── cava.js
    ├── clock.js
    ├── kb_layout.js
    ├── mixed.js
    ├── music.js
    ├── networkspeed.js
    ├── powerdraw.js
    ├── resources.js
    ├── simple_button.js
    ├── simple_clock.js
    ├── spaceleft.js
    ├── spaceright.js
    ├── system.js
    ├── tray.js
    ├── utils.js
    ├── window_title.js
    ├── workspaces_hyprland_focus.js
    ├── workspaces_hyprland.js
    ├── workspaces_sway_focus.js
    └── workspaces_sway.js

2 directories, 23 files

**Comment by PharmaRacist** [comment]
Uploading 2024-12-24 06-01-35.mp4…

what do you think of this thats not the final but trial inspired from novelknocks

**Reply by nichroot** [reply]
its not uploaded i think
